generic/tests/cfg/vlan.cfg

```diff
@@ -23,9 +23,8 @@
         vlan_num = 1
         win_vlan_id = 900
         cdroms += " virtio"
-        prepare_netkvmco_cmd = 'xcopy %s c:\\ /y && rundll32 netkvmco.dll,RegisterNetKVMNetShHelper'
-        set_vlan_cmd = 'netsh netkvm setparam 0 param=vlanid value=${win_vlan_id}'
         driver_verifier = netkvm
+        set_vlan_cmd = 'powershell -command "Set-NetAdapter -Name '%s' -VlanID '${win_vlan_id}' -Confirm:$False"'
```

Comment:
Can we remove the `'` char? The current line looks like contain 3 part:
`'powershell -command "Set-NetAdapter -Name '`%s`' -VlanID '`${win_vlan_id}`' -Confirm:$False"'`

Reply:
Got it, I will try to test it locally and will update after the test pass.